src/main/resources/env-map.yml

```diff
@@ -2,10 +2,10 @@ env_vars:
   BASE_URL:                                                                     '.[][] | select(.name == "app-catalog")               | .credentials.adminusers_url'
   SELFSERVICE_URL:                                                              '.[][] | select(.name == "app-catalog")               | .credentials.selfservice_url'
   SUPPORT_URL:                                                                  '.[][] | select(.name == "app-catalog")               | .credentials.card_frontend_support_url'
-  DB_HOST:                                                                      '.[][] | select(.name == "adminusers-db")             | .credentials.host'
-  DB_NAME:                                                                      '.[][] | select(.name == "adminusers-db")             | .credentials.name'
-  DB_PASSWORD:                                                                  '.[][] | select(.name == "adminusers-db")             | .credentials.password'
-  DB_USER:                                                                      '.[][] | select(.name == "adminusers-db")             | .credentials.username'
+  DB_HOST:                                                                      '.[][] | select(.name == "adminusers-secret-service") | .credentials.db_host'
+  DB_NAME:                                                                      '.[][] | select(.name == "adminusers-secret-service") | .credentials.db_name'
+  DB_PASSWORD:                                                                  '.[][] | select(.name == "adminusers-secret-service") | .credentials.db_password'
+  DB_USER:                                                                      '.[][] | select(.name == "adminusers-secret-service") | .credentials.db_user'
   DB_SSL_OPTION:                                                                '.[][] | select(.name == "adminusers-db")             | .credentials.ssl_option // "ssl=true"'
```

Comment:
This can probably be removed or added to the static values as `adminusers-db` will no longer exist. Or we can set `DB_SSL_OPTION: true` in the app manifest.